code/game/turfs/closed/wall/material_walls.dm

```diff
@@ -17,3 +17,7 @@
 	for(var/i in custom_materials)
 		var/datum/material/M = i
 		new M.sheet_type(src, FLOOR(custom_materials[M] / MINERAL_MATERIAL_AMOUNT, 1))
+
+/turf/closed/wall/material/mat_update_desc(mat)
+	desc = "A huge chunk of [mat] used to separate rooms."
```

Comment:
Mmmh alright I see some more use for this for these walls. However. You should remove the material flag and just run the proc, if the object doesn't update the desc based on materials thats up to them, the proc would just return.